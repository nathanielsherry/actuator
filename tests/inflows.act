#!/usr/local/bin/act

#This flowset should print "[['hello', 'world'], {'h2': 'hello', 'w2': 'world'}]" once

flow h from "hello" to @target;
flow w from "world" to @target;

flow target
  from inflow
  to @out
  on start;



flow h2 from "hello" to @named;
flow w2 from "world" to @named;

flow named
  from inflow(named)
  to @out;


flow out from inflow;
