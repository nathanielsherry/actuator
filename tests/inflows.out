[['hello', 'world'], {'h2': 'hello', 'w2': 'world'}]
